<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WasfurnBild v0.0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="toolTip">
         <string>Liest den Pfad des aktuellen Desktop-Hintergrundbildes aus der Registry.</string>
        </property>
        <property name="text">
         <string>Pfad auslesen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Kopiert den Pfad in die Zwischenablage.</string>
        </property>
        <property name="text">
         <string>Pfad kopieren</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelImage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Zeigt Desktop-Hintergrundbild, sobald Pfad ausgelesen wurde.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Öffnet das Bild im System-Bildbetrachter.</string>
        </property>
        <property name="text">
         <string>Bild öffnen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="toolTip">
         <string>Schließt das Programm.</string>
        </property>
        <property name="text">
         <string>Beenden</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
